contracts/hardhat.config.ts

```diff
@@ -31,7 +31,29 @@ const config: HardhatUserConfig = {
     path: abigen.bytecodeExporterPath,
     runOnCompile: true,
     clear: true,
+  },
+  namedAccounts: {
```

Comment:
Are these requirements from the deploy plugin ?

Reply:
Not required, but it's nice. Essentially allows you to declare which key belongs to which named account. Otherwise the plugin introduces it.